fpga/lib/ips/dino.cpp

```diff
@@ -141,23 +141,40 @@ void DinoAdc::setRegisterConfig(std::shared_ptr<Register> reg,
                                 double sampleRate) {
   constexpr double dinoClk = 25e6; // Dino is clocked with 25 Mhz
   constexpr size_t dinoRegisterTimer = 0;
-  constexpr size_t dinoRegisterScale = 1;
-  constexpr size_t dinoRegisterOffset = 2;
+  constexpr size_t dinoRegisterAdcScale = 1;
+  constexpr size_t dinoRegisterAdcOffset = 2;
+  constexpr size_t dinoRegisterDacExternalTrig = 4;
+  constexpr size_t dinoRegisterStsActive = 5;
+  constexpr size_t dinoRegisterDacScale = 6;
+  constexpr size_t dinoRegisterDacOffset = 7;
 
   uint32_t dinoTimerVal = static_cast<uint32_t>(dinoClk / sampleRate);
   double rateError = dinoClk / dinoTimerVal - sampleRate;
   reg->setRegister(
       dinoRegisterTimer,
       dinoTimerVal); // Timer value for generating ADC trigger signal
-  reg->setRegister(dinoRegisterScale,
-                   -0.001615254F);                // Scale factor for ADC value
-  reg->setRegister(dinoRegisterOffset, 10.8061F); // Offset for ADC value
-  uint32_t rate = reg->getRegister(0);
-  float scale = reg->getRegisterFloat(1);
-  float offset = reg->getRegisterFloat(2);
-  logging.get("Dino")->info("Check: Register configuration: Rate: {}, Scale: "
-                            "{}, Offset: {}, Rate-Error: {} Hz",
-                            rate, scale, offset, rateError);
+  reg->setRegister(dinoRegisterAdcScale,
+                   -0.001615254F); // Scale factor for ADC value
```

Comment:
Is there an equation on how this was calculated. Magic constants are not a nice code style..

Reply:
Would it be okay to move it to a configuration setting now? Maybe with your magic constant as a default value? I think it's crucial for future users to understand the meaning and how it was determined. So a sentence or two of documentation would not hurt here..

Reply:
I added a more detailed comment.